src/parsec_client/operations/psa_purge_key.md

```diff
@@ -0,0 +1,48 @@
+# PsaPurgeKey
```

Comment:
Similarly to the comment on your PR on `parsec-operations`, I think it is best to remove it from now.

Reply:
🧹